src/data/causes/cats.json

```diff
@@ -21,7 +21,8 @@
       "landing": {
         "title": "Open tabs.  Help shelter cats!",
```

Comment:
it looks like the copy has changed
https://www.figma.com/file/U69FDY32xoACHgIxyQaGD0/TFC-Brand-Visuals-Exploration?node-id=432%3A736